tests/cephfs/nfs/nfs_export_path.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from tests.cephfs.cephfs_volume_management import wait_for_process
+
+log = logging.getLogger(__name__)
```

Comment:
We recommend not using the default logger... instead opt to use the `Log` object in `cephci`.